.github/build.in.yml

```diff
@@ -183,6 +183,10 @@ jobs:
             lake build --no-build
           fi
 
+      - name: print the sizes of the oleans
```

Comment:
Can we move it before uploading the cache? I imagine we will want to short circuit if the oleans are too big

Reply:
Done!